schema/bls.mcf

```diff
@@ -15337,18 +15337,86 @@ rangeIncludes: schema:Number
 description: "A Consumer Price Index measures changes in the price level of a weighted average market basket of consumer goods and services purchased by households."
 descriptionUrl: "https://en.wikipedia.org/wiki/Consumer_price_index"
 
-Node: dcid:IndexPointBasePeriod1982To1984Equals100
-name: "IndexPointBasePeriod1982To1984Equals100"
+Node: dcid:IndexPointBasePeriodDecember2001Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 2001 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1983Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1983 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1988Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1988 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1982Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1982 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriod1967Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is 1967 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:InexPointBasePeriod1982To1984Equals100
```

Comment:
s/Inex/Index/g and check if the diffs look better (now it shows you removed a node when you're just trying to add others.

Reply:
fixed